lib/ansible/cli/doc.py

```diff
@@ -587,8 +575,45 @@ def get_man_text(self, doc):
             for note in doc['notes']:
                 text.append(textwrap.fill(CLI.tty_ify(note), limit - 6, initial_indent=opt_indent[:-2] + "* ", subsequent_indent=opt_indent))
             text.append('')
+            text.append('')
             del doc['notes']
 
+        if 'seealso' in doc and doc['seealso']:
+            text.append("SEE ALSO:")
+            for item in doc['seealso']:
+                if 'module' in item and 'description' in item:
+                    text.append(textwrap.fill(CLI.tty_ify('Module %s' % item['module']),
+                                limit - 6, initial_indent=opt_indent[:-2] + "* ", subsequent_indent=opt_indent))
+                    text.append(textwrap.fill(CLI.tty_ify(item['description']),
+                                limit - 6, initial_indent=opt_indent, subsequent_indent=opt_indent))
+                    text.append(textwrap.fill(CLI.tty_ify('https://docs.ansible.com/ansible/devel/modules/%s_module.html' % item['module']),
```

Comment:
Can we extract the version of ansible in use and use that rather than `/devel/` (or at least default to `/latest/`?

Reply:
+1 to this - we don't want to redirect users from the 2.5 docs to `devel` without any warning.

Reply:
Is there a variable that contains the version ?

Reply:
I reverted to using `latest`, as I don't see how we can get to the (major) version from ansible-doc at this time. We can improve this later if necessary.

PS I think using `/latest/` is probably better in any case, since we do specify when parameters are introduced and deprecated. And newer docs may have better information and examples.

Reply:
@acozine I think you misinterpret what is going on. The link is only exposed when using ansible-doc. So you won't be redirected from 2.5 docs to `devel` (now `latest`), because those references will go to the documentation you are using.

These links are only for people who run ansible-doc on the CLI. In that case we have to provide a link, and we cannot (assume) that a versioned-link exists for the command being used. So I think using `latest` is a good trade-off. Currently these references would simply fail to work. There would not be a link whatsoever, and that's what this proposal fixes too.